synapse/lib/cell.py

```diff
@@ -199,6 +199,10 @@ def setCellUser(self, iden):
         async for item in self.cell.dyniter(iden, todo, gatekeys=gatekeys):
             yield item
 
+    @adminapi
```

Comment:
Can we give this a perm as opposed to adminapi decorate it? admin users will automatically pass; and we can then allow mirror connections with non-admin users (in theory).

Reply:
this would essentially be the permissions to modify everything in the system. I'm not sure I see the value.